src/mame/mess.flt

```diff
@@ -401,6 +401,7 @@ interton/vc4000.cpp
 intv/intv.cpp
 isc/compucolor.cpp
 jazz/jazz.cpp
+juku/juku.cpp
```

Comment:
Yes, this seems reasonable, longer name of company/institute is [Arvutustehnika Erikonstrueerimisbüroo](https://et.wikipedia.org/wiki/Arvutustehnika_Erikonstrueerimisb%C3%BCroo), it's related to Estonian Academy of Sciences and EKTA is its trademark from 1980s. It's not completely clear to what extent there were companies in USSR and sometimes research and production had really interesting ways, but I think from computer architecture and reuse of technology point of view putting it under EKTA makes sense (also compared to other institutes in Estonia).

Reply:
Thanks, @happppp, I did notice, but I think it might be wise to consider it as a temporary category and still opt for separating mature drivers under institutes, product lines or schools of computing (BTW, it seems [Специалист](https://github.com/mamedev/mame/tree/master/src/mame/special) currently has escaped the category for some reason). For Juku the case is even more mixed because most kids using this computer at schools in the beginning of 1990s were already doing it in free Estonia and they would consider putting it under USSR label rather insulting. Also the engineers working to localise system for Estonian schools and language were building it precisely because political situation made it possible in the second half of 1980s and this is what individuates/motivates the Juku series. But I think it would be wrong also for more technical reasons to view computer production in USSR as a single category.

Reply:
@boamaod thanks for the heads up, specialist is now moved to proper location as well

Reply:
You're welcome, @mmicko, although it seems you have also missed [Agat](https://github.com/mamedev/mame/tree/master/src/mame/agat)!